scrapy_poet/autoextract.py

```diff
@@ -0,0 +1,87 @@
+from typing import ClassVar, Optional
+
+import attr
+from scrapy.http import Request
+from scrapy.settings import Settings
+from scrapy.statscollectors import StatsCollector
+
+from scrapy_poet.providers import PageObjectInputProvider
+
+from autoextract.aio.client import request_raw
+from autoextract_poet.page_inputs import (
+    ProductResponseData,
+    ArticleResponseData,
+)
+
+
+@attr.s(auto_attribs=True, slots=True)
+class Query:
```

Comment:
There is https://github.com/scrapinghub/scrapinghub-autoextract/blob/590e755710e3d4408b8ec1362e99a5f8af113363/autoextract/request.py#L10, which is supported by the library directly; I wonder if you can use it instead. `extra` support could be also added there.

Reply:
Documenting: this was moved to https://github.com/scrapinghub/scrapinghub-autoextract/pull/14/files